src/tests/unit/components/filter-input/FilterInput.spec.jsx

```
@@ -1,27 +1,45 @@
     expect(handleChange).toHaveBeenCalledWith('')
   })
+
+  it('calls updateFilter function on search icon (if used as button)', () => {
+    const handleChange = vi.fn()
+    render(<FilterInput value='' onChange={handleChange} />)
+  })
+
+  it('calls updateFilter function when Enter is pressed', () => {
+    const handleChange = vi.fn()
+    const { getByRole } = render(
+      <FilterInput value='typed' onChange={handleChange} />
```

ESLint

[error] Props should be sorted alphabetically

(react/jsx-sort-props)